pkg/spanconfig/spanconfigreconciler/reconciler.go

```diff
@@ -560,7 +624,13 @@ func (r *incrementalReconciler) reconcile(
 					return err
 				}
 
-				translator := r.sqlTranslatorFactory.NewSQLTranslator(txn)
+				// Due to the fact that we are reading descriptors at a fixed historical
+				// timestamp, the dropping of invalid PTS records in the previous step
```

Comment:
hoping none of this is necessary, given my point above about the full reconciliation being sufficient.

Reply:
you make a good point here. @kev-cao i think we should put this PR on ice for now, since we have addressed main PCR problem.